src/js/actions/sampler.js

```diff
@@ -409,17 +409,15 @@ define(function (require, exports) {
         var uiStore = this.flux.store("ui"),
             docStore = this.flux.store("document"),
             coords = uiStore.transformWindowToCanvas(x, y),
-            layerTree = doc.layers;
+            layers = doc.layers;
```

Comment:
I'm not saying I love the `layerTree` var name, but just pointing out that I think it has been used in the past to differentiate the "layer structure" from a simple list of layers.